service-tests/github.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'github', title: 'GitHub' });
+module.exports = t;
+
+t.create('downloads for release without slash')
+  .get('/downloads/atom/atom/v0.190.0/total.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('downloads'),
+    value: Joi.not('none')
```

Comment:
`Joi.not('none')` will accept a lot of garbage values, so I'd like to see something more specific. @webcaetano just wrote a [nice regex for file size](https://github.com/badges/shields/pull/745/files#diff-cf32c23424cd5f1a16820ff81b14d833R14). Could you adapt that?